core/src/main/resources/lib/hudson/buildListTable.jelly

```diff
@@ -45,7 +45,8 @@ THE SOFTWARE.
                       insert(generateSVGIcon(e.iconName, "${iconSizeClass}"))));
               tr.insert(new Element('td').
                   insert(new Element('a', {class: 'jenkins-table__link model-link', href: '${rootURL}/' + e.parentUrl}).
-                      update(e.parentFullDisplayName)).
+                   insert(new Element('span').
```

Comment:
This part improves the break lines in the "Build history page".

### Before
![Before](https://user-images.githubusercontent.com/71805759/169386732-ddb779c8-6fe9-4a48-83cf-97dd4c2bdf4b.PNG)

### After
![After](https://user-images.githubusercontent.com/71805759/169386785-60e78042-0e53-4829-a337-85aee74acf64.PNG)